src/features/AddAccount/stories.tsx

```diff
@@ -53,23 +54,35 @@ export const getStories = (): IStory[] => [
   },
   {
     name: WalletId.KEYSTORE_FILE,
-    steps: [NetworkSelectPanel, IS_DEV || IS_ELECTRON ? KeystoreDecrypt : InsecureWalletWarning],
-    isDisabled: IS_NOT_ELECTRON_AND_IS_NOT_DEV
+    steps: [
+      NetworkSelectPanel,
+      IS_DEV || IS_STAGING || IS_ELECTRON ? KeystoreDecrypt : InsecureWalletWarning
+    ],
+    isDisabled: IS_NOT_ELECTRON_AND_IS_NOT_DEV_OR_STAGING
```

Comment:
We got to be careful that this doesn't get out of hand.
1. Staging is suppose to mirror Production. If the feature set or build process start to deviate from each other, then the whole point of staging becomes useless.
- Can we use a feature toggle to enable it for tests? (eg. PTX test in `send.test.js`)
- Or can we run the tests just for electron builds? 
2. The double negation confuses me:
` isDisabled: IS_NOT_ELECTRON_AND_IS_NOT_DEV_OR_STAGING`
can we simplify to something like? 
```
isDisabled: IS_WEB && IS_PROD
//or

isEnabled: IS_ELECTRON || !IS_PROD
```